content/spin/v3/dynamic-configuration.md

```diff
@@ -260,9 +262,25 @@ container = "<cosmos-container>"
 
 > Note: The CosmosDB container must be created with the default partition key, `/id`.
 
-Whilst a single default store may be sufficient for certain application use cases, each Spin application can be configured to support multiple stores of any `type`, as shown in the `runtime-config.toml` file below:
+### AWS DynamoDB Key Value Store Provider
+
+To use an Amazon Web Services DynamoDB database as a backend for Spin's key/value store, change the type to `aws_dynamo` and specify your database account details:
+
+```toml
+[key_value_store.default]
+type = "aws_dynamo"
+region = "<aws-region>"   # e.g. "us-east-1"
+table = "<dynamo-table>"  # e.g. "spin-key-values"
+consistent_read = true    # depending on your consistency requirement
```

Comment:
It might be worth noting that setting this true will double your read capacity unit (RCU) usage -- generally doubling costs. Though perhaps users setting this would already be familiar with the extra cost that consistent reads entail

Reply:
Added note with link to the AWS consistent reads page